From 4b924e573da307436169d5ef7e04c0ab85b36ef9 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@intel.com>
Date: Wed, 2 Aug 2017 15:17:21 +0100
Subject: [PATCH] tld-parser: use Python 3

The world is moving to Python 3 so do our bit by using Python 3 to run the TLD
parser.

https://bugzilla.gnome.org/show_bug.cgi?id=785735
---
 libsoup/tld-parser.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libsoup/tld-parser.py b/libsoup/tld-parser.py
index 5d9d2ba5..a743471a 100755
--- a/libsoup/tld-parser.py
+++ b/libsoup/tld-parser.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 
 # Generate tld rules
 # Copyright (C) 2012 Red Hat, Inc.
-- 
2.18.1

